app/View/Pages/series.ctp

```diff
@@ -0,0 +1,100 @@
+<?php $this->set('title_for_layout', Configure::read("Club.name")); ?>
+
+<div class="three-column row">
+    <article class="col-sm-4 col-sm-push-4 events-by-series">
+        <header>
+          <h2>Events</h2>
+        </header>
+        <?php
+        $events = array();
+        $ongoing = $this->element('Events/box-list', array('filter' => 'ongoing', 'limit' => '-1'));
+
+        if(!empty($ongoing)) {
+            echo "<h3>${title}</h3>";
+            echo $ongoing;
+        }
+
+        $seriesSet = $this->requestAction('series/index');
+
+        $upcoming = array();
+        foreach($seriesSet as $series) {
+            if (!$series["Series"]["is_current"])
```

Comment:
hmm, I can't remember what this `is_current` is used for. Can we just remove this check as it doesn't seem too necessary here? (and might prevent some content from showing up in the case that someone didn't set up their series correctly)

Reply:
Oh great, thanks for the context! I think checking `is_current` makes sense then.